### ISSUE TITLE
python version? 

### ISSUE THREAD

**Issue by arademaker** [nigelm/moin2gitwiki#1]
Does it needs Python 2 or 3? Not clear. In MacOS, any specific instruction how to prepare the environment? I am getting the messages:

```
% poetry run moin2gitwiki --moin-data ~/hpsg/documentation/docs/ltg/moin/delphin/data --user-map users.delphin fast-export  ~/Temp/doctest2
....
2021-04-29 19:07:40,326 — moin2gitwiki — WARNING — fetch:93 — No response to http://localhost/jrtwiki/ParticipantsGroup?action=recall&rev=00000065
2021-04-29 19:07:40,370 — moin2gitwiki — WARNING — fetch:93 — No response to http://localhost/jrtwiki/ParticipantsGroup?action=recall&rev=00000066
2021-04-29 19:07:40,415 — moin2gitwiki — WARNING — fetch:93 — No response to http://localhost/jrtwiki/PetOutput?action=recall&rev=00000001
2021-04-29 19:07:40,461 — moin2gitwiki — WARNING — fetch:93 — No response to http://localhost/jrtwiki/PetOutput?action=recall&rev=00000002
...
```

But so many `no response...`

**Comment by arademaker** [comment]
In the final repository, I got all files with 1 byte and empty content...

**Comment by dehylton** [comment]
following the "fastexport" command you need to supply the "--url-prefix http://my.wiki.here" option, as the page itself is retrieved via http ...

having said that, doing so in my case solved the "no response" errors but still resulted in blank output.

**Comment by nigelm** [comment]
Sorry - had not been seeing emails from github (although I found one in spam).

Python 2 has been obsolete for way over a year - I don't think it should even need specifying for new code that its going to be python 3.  And the `pyproject.yaml` file has python 3.8 as a dependancy.

I also found that I had a set of changes that had not been pushed to github - these are now in place.

Hopefully the combination of those two things helps.

**Comment by nigelm** [comment]
Sorry - had not been seeing emails from github (although I found one in spam).

Python 2 has been obsolete for way over a year - I don't think it should even need specifying for new code that its going to be python 3.  And the `pyproject.yaml` file has python 3.8 as a dependancy.

I also found that I had a set of changes that had not been pushed to github - these are now in place.

Hopefully the combination of those two things helps.
